marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -401,9 +402,9 @@ public void clearUserModules(List<String> resourceNamesToNotDelete) {
                     "] ! xdmp:document-delete(.)\n";
             runInDatabase(query, hubConfig.getDbName(DatabaseKind.MODULES));
         } catch (Exception e) {
-            logger.error("Failed to clear user modules, cause: " + e.getMessage(), e);
+            throw new RuntimeException(e.getMessage());
```

Comment:
Let's keep the message though that was being logged, but just put it in the exception - e.g. throw new RuntimeException("Failed to clear user modules, cause: " + e.getMessage(), e).